README.md

```diff
@@ -0,0 +1,32 @@
+# Sitecore Headless Samples and Examples
+This repository provides Sitecore and community-developed examples of various implementation and integration scenarios, for implementations which utilize Sitecore Headless SDKs. While Sitecore already provides official boilerplate and starters for these frameworks, questions about how to do X or Y often come up in community channels. The aim for this repository is to build a library of samples to help address these questions and provide a reference for Sitecore developers.
```

Comment:
Sounds a little wordy with "implementation" repeated. Also we're staying away from "starters", right? (Friends don't let friends use boilerplate as a starter) How about...

```suggestion
This repository provides various examples of Sitecore Headless SDKs implementations provided by Sitecore and community developers. While Sitecore already provides official boilerplate and samples for the different frameworks supported by the Headless SDKs, sometimes questions about use cases not covered by these samples pop up multiple times in community channels. The aim for this repository is to build a library of samples to help address these common questions and provide a reference for Sitecore developers.
```